src/chainstate/stacks/index/bits.rs

```diff
@@ -233,8 +224,9 @@ pub fn get_nodetype_hash_bytes<T: MarfTrieId, M: BlockMap>(
 
 /// Low-level method for reading a TrieHash into a byte buffer from a Read-able and Seek-able struct.
 /// The byte buffer must have sufficient space to hold the hash, or this program panics.
+#[inline]
```

Comment:
Are all these `#[inline]` directives helpful? It's usually better to let the compiler make a decision on inlining (it's actually going to end up making the decision anyways), as the trade-off isn't always obvious. And unless it's really clear that these are helpful, I think it's better practice to leave these kinds of directives out.